spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/actions/TestSparkFileRewriter.java

```diff
@@ -302,6 +305,16 @@ public void testInvalidValuesForBinPackDataOptions() {
         ImmutableMap.of(SizeBasedDataRewriter.DELETE_FILE_THRESHOLD, "-1");
     assertThatThrownBy(() -> rewriter.init(invalidDeleteThresholdOptions))
         .hasMessageContaining("'delete-file-threshold' is set to -1 but must be >= 0");
+
+    Map<String, String> negativeDeleteRatioThresholdOptions =
+        ImmutableMap.of(SizeBasedDataRewriter.DELETE_RATIO_THRESHOLD, "-1");
+    assertThatThrownBy(() -> rewriter.init(negativeDeleteRatioThresholdOptions))
+        .hasMessageContaining("'delete-ratio-threshold' is set to negative but must be >= 0");
```

Comment:
this (and other places) should also have a `.isInstanceOf(IllegalArgumentException.class)` right before this line. We generally want to make sure that the right type of exception is thrown